test/unit/presenters/publishing_api_presenters/topical_event_about_page_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'test_helper'
+
+class PublishingApiPresenters::TopicalEventAboutPageTest < ActiveSupport::TestCase
+  def present(record)
+    PublishingApiPresenters::TopicalEventAboutPage.new(record)
+  end
+
+  test "topical event about page presentation includes the correct values" do
+    topical_event_about_page = create(:topical_event_about_page, content_id: SecureRandom.uuid)
```

Comment:
Indeed! And I had. Removed the content_id setting now [here](https://github.com/alphagov/whitehall/commit/5614dba2632363437761ba5e45f2a2f4b62812c8#diff-fb5206241c7dbc041de33c431304e018R9).